kubernetes/ansible/roles/sunbird-monitoring/templates/prometheus-operator.yaml

```diff
@@ -11,10 +11,12 @@ fullnameOverride: sunbird-monitoring
 # Enabling external prometheus scrape config 
 prometheus:
   prometheusSpec:
-    additionalScrapeConfigsExternal: {{ additional_scrape_configs_enabled | d(true) }}
+    thanos:
```

Comment:
How can we push thanos without review?

Reply:
thanos is just a side car, which is running over there. So that we can measure whether it's using any more resources. it's kinda like a feature flag, just running but not using that

Reply:
This is running in load test for a while, and then but with full fledged. this can be consider as first step.